web/src/engine/websites/TuMangaOnline.d.ts

```diff
@@ -0,0 +1,7 @@
+import type { Page } from '../providers/MangaPlugin';
```

Comment:
Just for knowledge purpose, whats the point of this file? Should i do the same for https://github.com/manga-download/haruneko/pull/528 ?

Reply:
After compilation and obfuscation there is only a JavaScript file. This file exposes the type definitions which are missing in the JavaScript file for better integration in a TypeScript project.

This file is automatically created when executing `npm run deploy:prod` (assuming the relative paths of haruneko and haruneko-proprietary are in the same parent directory).

It is strongly recommend to add this file.

Reply:
Gotta do that for mangadig then, thanks